lua/indentmini/init.lua

```diff
@@ -34,7 +34,7 @@ local function indentline(hl_current)
 
     ctx[row] = indent
 
-    for i = 1, indent - 1, vim.bo[bufnr].sw do
+    for i = 1, indent - 1, vim.fn.shiftwidth() do
```

Comment:
can there add a check like `vim.fn.exist('*shiftwidth) == 1 ` ?

Reply:
Done, but I just realized that `vim.opt/opt_local.shiftwidth:get()` sometimes returns `0`. That might be why the loop was running forever in the first place. so I suggest when `shiftwidth fn` doesn't exist and `shiftwidth:get()` is returning `0`, we use the default neovim width of `8`. what do you think.

Reply:
@glepnir , try this [.editorconfig](https://github.com/Ultra-Code/awesome-neovim/blob/master/.editorconfig) without this pr to experience the nvim hang issue due to shiftwidth been `0`. I sense you don't like the 😞 `fn.shiftwidth` impl in the `elseif` branch. If its a real big issue for you, I could remove it.